submissions/aymkin/HTML-CSS-Popup/index.html

```diff
@@ -93,13 +93,14 @@
             </li>
             <li class="nav__element tiel">
                 <input
-                    type="checkbox"
-                    class="nav__element pop-check"
-                    name="tiel"
+                type="checkbox"
+                class="nav__element pop-check"
+                name="tiel"
                 />
+                <div class="outline"></div>
```

Comment:
What purpose of this `div`?

Reply:
That was a workaround for hover and focus states. That is deleted and refactored.